lib/invoker/dns_cache.rb

```diff
@@ -2,22 +2,84 @@ module Invoker
   class DNSCache
     attr_accessor :dns_data
 
+    # INI:
+    # [app]
+    # port = 8000
+    # location = www2.app/blah
+    #
+    # [chat]
+    # port = 8001
+    # location www.app/chat
+    #
+    # [api]
+    # port = 8002
+    # location = www.app/api
+    #
+    # dns_data:
+    # {
+    #   'api' => [
+    #     ['', 8002]
+    #   ],
+    #   'app' => [
+    #     ['', 8000]
+    #   ],
+    #   'chat' => [
+    #     ['', 8001]
+    #   ],
+    #   'www.app' => [
+    #     ['', 8000],
+    #     ['/api, 8002],
+    #     ['/chat, 8001]
+    #   ],
+    #   'www2.app' => [
+    #     ['/blah', 8000]
+    #   ]
```

Comment:
I'm not sure if you're referring to the example INI or the `dns_data` object, but the INI is referring to the new format with `location`.

The `dns_data` object is simply an internal object representation.